### DISCUSSION TITLE
New time for community meetings?

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#433] [General]
I've been swamped with beginning-of-semster and impending-NSF-deadline business the last few weeks, so I've neglected in organizing our next community call.

I'm fairly busy on Friday afternoons this semester, so I wanted to see if our regulars would be open to a different time for our community call. Either Monday or Wednesday at the usual time (3pm Central), or anytime Mon/Wed afternoon really, works well for me.

Thoughts?

**Comment by ccaruvana** [comment]
Yeah, the beginning of the semester has slammed my time, as well. Somewhat unfortunately, my Mondays-Thursdays are more or less full from morning to afternoon, but I could probably do Wednesdays during my lunch hour+ (11:30am - 1:30pm Eastern).

**Reply by StevenClontz** [reply]
I'm also flexible on Fridays until the early afternoon if that's helpful

**Reply by ccaruvana** [reply]
Yeah, Fridays are generally pretty clear for me!

**Reply by prabau** [reply]
@StevenClontz what time are you available/not available on Fridays?

**Reply by StevenClontz** [reply]
I start office hours at 130 Central and am flexible before that.

**Reply by prabau** [reply]
1:30 Central = 2:30 Eastern (I live in NY). So no problem for me, you guys just pick a time.  And we could also make it less than an hour if needed.

**Reply by StevenClontz** [reply]
Starting sometime 12noon-1:30pm Eastern works for me. @lyengulalp how are things on your end?

**Comment by lyengulalp** [comment]
I can do Fridays at 12 eastern!
